dev/CommonStyles/Slider_themeresources.xaml

```diff
@@ -174,14 +174,15 @@
         </ResourceDictionary>
     </ResourceDictionary.ThemeDictionaries>
 
+    <Thickness x:Key="SliderContainerMargin">2</Thickness>
     <Thickness x:Key="SliderTopHeaderMargin">0,0,0,4</Thickness>
     <CornerRadius x:Key="SliderTrackCornerRadius">2</CornerRadius>
     <CornerRadius x:Key="SliderThumbCornerRadius">10</CornerRadius>
-    <x:Double x:Key="SliderPreContentMargin">15</x:Double>
-    <x:Double x:Key="SliderPostContentMargin">15</x:Double>
+    <x:Double x:Key="SliderPreContentMargin">13</x:Double>
+    <x:Double x:Key="SliderPostContentMargin">13</x:Double>
     <x:Double x:Key="SliderInnerThumbHeight">12</x:Double>
-    <x:Double x:Key="SliderHorizontalHeight">32</x:Double>
-    <x:Double x:Key="SliderVerticalWidth">32</x:Double>   
+    <x:Double x:Key="SliderHorizontalHeight">30</x:Double>
+    <x:Double x:Key="SliderVerticalWidth">30</x:Double>   
```

Comment:
Unfortunately, due to Slider not having different normal visual states for Vertical and Horizontal orientation, introducing margin will change the default size of the entire control. To mitigate that, I've adjusted Heights/Widths as well as content margins.

Reply:
And although the actual size of the control doesn't change, the visual of Slider will shrink by 2px from each side.